# std_msgs/Header header
float64[7] currentJoint
string targetPoint
---
bool finishPublishCommand